integration-tests/stdin-context.test.ts

```diff
@@ -67,4 +67,31 @@ describe('stdin context', () => {
       'Expected the model to identify the secret word from stdin',
     ).toBeTruthy();
   });
+
+  it.only('should exit quickly if stdin stream does not end', async () => {
```

Comment:
@dumbbellcode thank you for the contribution! Would you mind removing this `.only` portion? It'll skip all the other test sotherwise